src/sidebar/components/stream-content.js

```diff
@@ -59,9 +59,6 @@ function StreamContentController(
   fetch(20);
 
   this.setCollapsed = store.setCollapsed;
-  this.forceVisible = function(id) {
```

Comment:
I find this implementation confusing and I'm not sure how it was working as expected before?

Reply:
It looks like this was indeed broken, given that the corresponding method in `<sidebar-content>` expects a thread rather than a thread ID.